source/server/connection_handler_impl.cc

```diff
@@ -313,23 +313,24 @@ ConnectionHandlerImpl::getBalancedHandlerByAddress(const Network::Address::Insta
   // Otherwise, we need to look for the wild card match, i.e., 0.0.0.0:[address_port].
   // We do not return stopped listeners.
   // TODO(wattli): consolidate with previous search for more efficiency.
-  std::string addr_str =
-      address.ip()->version() == Network::Address::IpVersion::v4
-          ? Network::Utility::getIpv4AnyAddress(address.ip()->port())->asString()
-          : Network::Utility::getIpv6AnyAddress(address.ip()->port())->asString();
+
+  std::string addr_str = address.ip()->version() == Network::Address::IpVersion::v4
+                             ? Network::Address::Ipv4Instance(address.ip()->port()).asString()
+                             : Network::Address::Ipv6Instance(address.ip()->port()).asString();
```

Comment:
Why not use `Utility::getIpv4AnyAddress` any more? It seems more efficient than build an new any address Instance.